CHANGELOG.md

```diff
@@ -5,21 +5,29 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [v1.0.0]
+##[Unreleased]
 
-This version is compatible with [SimpleSAMLphp v1.14](https://simplesamlphp.org/docs/1.14/simplesamlphp-changelog)
+###Added
 
-### Added
+- Support for parsing email address
 
-- FullNameParser library
-  - PHP library to split names into their respective components (first, last, etc)
-- FullNameParsing class
-  - Module class that uses the library
+### Fixed
 
-## [v1.0.1]
+- Fix attribute names in the condition of FullNameParsing class
```

Comment:
Replace with 'Fix check of existing first/last name information in user attributes based on configured attribute names'

Reply:
Done